docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -250,65 +227,419 @@ access_entries = {
 
-Terraform can define outputs to make the retrieval of values generated as part of the execution easier; for example, DB endpoints or values required for the Helm setup.
+We separated the cluster and PostgreSQL modules to offer you more customization options.
 
-1. In the folder where your `config.tf` resides, create an additional `output.tf`.
-2. Paste the following content to expose those variables:
+#### Step 1: create a configuration file for the database
 
-```hcl
-output "cert_manager_arn" {
-  value = module.eks_cluster.cert_manager_arn
-  description = "The Amazon Resource Name (ARN) of the AWS IAM Roles for Service Account mapping for the cert-manager"
-}
+1. In the directory where your `config.tf` file resides, create a new file named `db.tf`.
+2. Add the following content to `db.tf` to use the provided PostgreSQL module:
 
-output "external_dns_arn" {
-  value = module.eks_cluster.external_dns_arn
-  description = "The Amazon Resource Name (ARN) of the AWS IAM Roles for Service Account mapping for the external-dns"
-}
+<Tabs groupId="db-tf">
+  <TabItem value="standard" label="Standard" default>
 
-output "postgres_endpoint" {
-  value = module.postgresql.aurora_endpoint
-  description = "The Postgres endpoint URL"
-}
+```hcl reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/db.tf
+```
+
+  </TabItem>
+  <TabItem value="irsa" label="IRSA">
+
+In addition to using standard username and password authentication, you can opt to use [**IRSA (IAM Roles for Service Accounts)**](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/) for secure, role-based access to your Aurora database. This method allows your EKS workloads to assume IAM roles without needing to manage AWS credentials directly.
+
+**Note**: Using IRSA is optional. If preferred, you can continue using traditional password-based authentication for database access.
+
+If you choose to use IRSA, you’ll need to take note of the **IAM role** created for Aurora and the **AWS Account ID**, as these will be used later to annotate the Kubernetes service account.
+
+##### Aurora IRSA role and policy
+
+The Aurora module uses outputs from the EKS cluster module to configure the IRSA role and policy. Below are the required parameters:
+
+Here’s how to define the IAM role trust policy and access policy for Aurora:
+
+```hcl reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6-irsa/db.tf
+```
+
+Once the IRSA configuration is complete, make sure to **record the IAM role name** (from the `iam_aurora_role_name` configuration) and the **AWS Account ID** (from `module.eks_cluster.aws_caller_identity_account_id`), as these will be required to annotate the Kubernetes service accounts during the helm configuration.
```

Comment:
what for do you need the `aws_caller_identity_account_id` in the following?
At least for this document I didn't find the usage.